po/de.po

```diff
@@ -1839,7 +1838,8 @@ msgstr ""
 "(Format: \"calc:xxx\")\n"
 "  14. eine Option (Format: \"file.section.option\")\n"
 "  15. der Name einer lokalen Variablen eines Buffer\n"
-"  16. ein hdata Name/Variable (der Wert wird automatisch in eine "
```

Comment:
The item number 9 is now a modifier, it seems this is not present and then items after this one are shifted.

Reply:
ok i've fixed point 9 (and some other items)